### ISSUE TITLE
S3::getSignedUrl should error when passing a non-integer to Expire

### ISSUE THREAD

**Issue by jwerre** [aws/aws-sdk-js#2648]
I noticed that when you pass a String to the `S3::getSignedUrl` `Expires` parameter an expired url is produced. 

```js

	let options = {
		Bucket: BUCKET,
		Key: KEY
	};

	options.Expires = 259200; // this works as expected.
	options.Expires = '259200'; // this works as as well but the presigned url is expired.
	
	s3.getSignedUrl('getObject', options, function (err, url) {
		if (err) {
			return throw err;
		}
		console.log(url);
	});
```

It would be nice if an error was produced if `Expires` is not a Number. I was passing it in from a Lambda environmental variable and had a heck of a time debugging this.

**Comment by AllanFly120** [comment]
Hi @jwerre,

Thank you for reporting this. We should raise error when `Expires` is not a number in `s3.getSignedUrl`.

**Comment by trivikr** [comment]
Fixed in #2668
